docs/book/src/development/development.md

```diff
@@ -156,6 +169,8 @@ Make sure that the image or sysext overlay you use exists for and works with tha
 **Note:** All clusters created from the dev-test ClusterClass will require a secret named `dev-test-cloud-config` with the `clouds.yaml` to be used by CAPO for interacting with OpenStack.
 You can create it manually or see below how to make Tilt automate it.
 
+**Security Note:** When using `allowedCIDRs`, ensure you include the IP range of your management cluster to maintain CAPO connectivity. When enabling a bastion host, configure appropriate SSH key access and firewall rules.
+
```

Comment:
I think we already have this documented in the "book" and would say that it is unnecessary in the development notes.

```suggestion
```

Reply:
Found it in configuration.md

Removed.